app/services/sla_breach_service.rb

```diff
@@ -10,7 +10,12 @@ def breach_threshold
 
   # attention_needed_since is present and is before breach threshold
   # returns data in the format { vita_partner_id: breach_count }
-  def attention_needed_breach
+  def attention_needed_breaches
     Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:attention_needed_since].lteq(breach_threshold)).count(:id)
   end
+
+  # clients who messaged us and have not been responded to _with a message, call or email_ within the breach threshold
+  def outgoing_communication_breaches
+    Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:first_unanswered_incoming_correspondence_at].lteq(breach_threshold)).count(:id)
```

Comment:
It took me a second to understand how being less than a threshold would lead to a breach. Then I realized that we were comparing dates.. maybe calling it breach_threshold_date would be clearer?